File: Python/day_three.py
def most_frequent_char(s):
  characters = {}
  for letter in s:
    if letter not in characters:
      characters[letter] = 1
    else:
      characters[letter] += 1
  max_value = 0
  max_key = ""
  for key, value in characters.items():
    if value > max_value:
      max_value = value
      max_key = key
  return max_key

File: JS/dayThree.js
const mostFrequentChar = (s) => {
    const character = {};
    for (let letter of s) {
      if (character[letter] === undefined) {
        character[letter] = 1;
      } else {
        character[letter] += 1;
      }
    }
    let maxValue = 0;
    let maxKey = "";
    for (let key in character) {
      if (character[key] > maxValue) {
        maxValue = character[key];
        maxKey = key;
      }
    }
    return maxKey;
  };


  const pairSum = (numbers, targetSum) => {
    for (let x = 0; x < numbers.length - 1; x++) {
      for (let y = 1; y < numbers.length; y++) {
        if (numbers[x] + numbers[y] == targetSum) {
          return [x, y]
        }
      }
    }
  };
